### PR TITLE
Bufix/v3/order seeder item

### PR DESCRIPTION
**Pull request by afdal-develop**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,15 @@
-MIT License
```
Comment (rizkiheryandi) [inline LICENSE:1]: Why changes this file ?

File: .github/FUNDING.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
 # These are supported funding model platforms
```
Comment (rizkiheryandi) [inline .github/FUNDING.yml:1]: Why changes this file ?

File: README.md [hunk 3]
```diff
@@ -10,8 +10,8 @@
```
Comment (rizkiheryandi) [inline README.md:10]: Why changes this file ?

File: LICENSE (hunk 1 above)
Comment (afdal-develop) [reply LICENSE]: sorry, i can rollback the code

File: .github/FUNDING.yml (hunk 2 above)
Comment (afdal-develop) [reply .github/FUNDING.yml]: sorry, i can rollback the code

File: README.md (hunk 3 above)
Comment (afdal-develop) [reply README.md]: sorry, i can rollback the code

**Pull request closed without merging**